ray-operator/controllers/ray/common/pod.go

```diff
@@ -2,11 +2,13 @@ package common
 
 import (
 	"bytes"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"strconv"
 	"strings"
 
+	"github.com/alessio/shellescape"
```

Comment:
Used by KinD and other K8s SIGs, so should be legit enough for us?
https://pkg.go.dev/github.com/alessio/shellescape?tab=importedby

Reply:
I think so.